src/server/debugcmd.cc

```diff
@@ -361,20 +365,22 @@ void DebugCmd::PopulateRangeFiber(uint64_t from, uint64_t num_of_keys,
   SetCmd::SetParams params;
 
   uint64_t index = from;
+  uint64_t to = from + num_of_keys;
   uint64_t added = 0;
   while (added < num_of_keys) {
-    if ((index > from) && (index % num_of_keys == 0)) {
+    if ((index >= to) && ((index - to) % options.total_count == 0)) {
       index = index - num_of_keys + options.total_count;
```

Comment:
shouldn't we update `to` at this point now?

Reply:
I believe I dont.
I want to check if 
index = total_count * i + from + number_of_keys
to is set to  from + number_of_keys

Reply:
right!